tf2onnx/tfonnx.py

```diff
@@ -690,6 +690,18 @@ def sign_op(ctx, node, name, args):
     return nodes
 
 
+def sign_op9(ctx, node, name, args):
+    # Currently supported: `float32`
+    # Ignored: `bfloat16`
+    # TODO: add support for `int32`, `int64`
```

Comment:
i mean you can (optionally) add a TODO at test side. :-)
just remove these 3 lines and keep the test as it is.

you can fix the cast issue we mentioned today and update test with more types in another PR.